worlds/subnautica/__init__.py

```diff
@@ -34,12 +32,30 @@ class SubnauticaWorld(World):
     game: str = "Subnautica"
     web = SubnaticaWeb()
 
-    item_name_to_id = items_lookup_name_to_id
-    location_name_to_id = locations_lookup_name_to_id
-    options = options
+    item_name_to_id = {data["name"]: item_id for item_id, data in Items.item_table.items()}
+    location_name_to_id = {data["name"]: loc_id for loc_id, data in Locations.location_table.items()}
+    options = Options.options
 
     data_version = 2
-    required_client_version = (0, 1, 9)
+    required_client_version = (0, 3, 3)
+
+    prefill_items: List[Item]
+
+    def generate_early(self) -> None:
+        self.prefill_items = [
+            self.create_item("Seaglide Fragment"),
+            self.create_item("Seaglide Fragment")
+        ]
+
+    def create_regions(self):
+        self.world.regions += [
+            self.create_region('Menu', None, ['Lifepod 5']),
+            self.create_region('Planet 4546B',
+                               Locations.events + [location["name"] for location in Locations.location_table.values()])
+        ]
+
+    def set_rules(self):
+        set_rules(self.world, self.player)
```

Comment:
this is confusing to a reader
maybe `from . import Rules; Rules.set_rules(self.world, self.player)` would be better?